Library/premake5.lua

```diff
@@ -206,17 +206,14 @@ if not Project then
             defines { 
                 "CSP_IOS",
                 "USE_STD_MALLOC=1",
-                "JS_STRICT_NAN_BOXING"
+                "JS_STRICT_NAN_BOXING",
+                "USE_STD_MALLOC=1"
```

Comment:
You have "USE_STD_MALLOC" repeated.

Reply:
Thank you for saving me from myself 😅 Fixed in latest!